docs/spikes/azure-sphere.md

```diff
@@ -0,0 +1,90 @@
+# What is Azure Sphere
+
+[Azure Sphere](https://azure.microsoft.com/en-us/services/azure-sphere/) is a security service
+combining hardware, OS-level software, and cloud services to secure IoT applications. Together these
+three components ensure [The Seven Properties of Highly Secure
+Devices](https://www.microsoft.com/en-us/research/uploads/prod/2020/11/Seven-Properties-of-Highly-Secured-Devices-2nd-Edition-R1.pdf).
+
+[Pricing](https://azure.microsoft.com/en-us/pricing/details/azure-sphere/): one time purchase of
+hardware only (no recurring fees)
+
+## Advantages for the starter kit
+
+- Automated management of the underlying OS and the concentrator logic.
+- Although some features are already supported e.g. password-less authentication, we could benefit
+  from built-in, e.g:
+  - hardware security
+  - error reporting (i.e. A software error, such as a buffer overrun induced by an attacker probing
+security, is reported to cloud-based failure analysis system)
+  - renewable security (revoking compromised assets for known vulnerabilities)
+- Cost savings: considering the maintenance required for the concentrators, the additional or more expensive
+  hardware could be justified or even be cheaper in the long term.
+
+## Disadvantages
+
+- Requires specific hardware with [limited choices so
+  far/potentially more expensive](https://azure.microsoft.com/en-us/services/azure-sphere/#ecosystem) (some also out of stock)
+  - There is a [LoRaWAN concentrator built from
+    Miromico](https://www.avnet.com/wps/portal/silica/solutions/technologies/wireless-connectivity/lora-gateways/)
+    that only works with the Packet Forwarder for now. Miromico are aware of this limitation but we
+    are not sure when they plan to support LBS.
+- Introduces another layer that needs to be considered.
+
+## Process for the concentrators
+
+[Quickstart](https://docs.microsoft.com/en-us/azure-sphere/install/overview)
+
+- Get the hardware (either a new device or "guardian module" that is "attached" to existing hardware)
+- Install SDK/CLI + [VS
+  extension](https://marketplace.visualstudio.com/items?itemName=AzureSphereTeam.AzureSphereSDKforVisualStudio2019)
+  and connect the device to the PC
+- [Create a
+  "tenant"](https://docs.microsoft.com/en-us/azure-sphere/deployment/manage-tenants?tabs=cliv2beta)
+- [Claim the
+  device](https://docs.microsoft.com/en-us/azure-sphere/install/claim-device?tabs=cliv2beta):
+  one-time operation that can not be undone
+- [Configure networking](https://docs.microsoft.com/en-us/azure-sphere/install/configure-wifi)
```

Comment:
[nit] Instead of duplicating the steps from the Azure Sphere quickstart, which might get outdated potentially, I think it's good enough to just link to the official quickstart.

If you think it makes sense to capture what the steps were at the time of writing, it's also fine to resolve the comment.

Reply:
Fair point but because we had some notes that are specific to us, I think the short recap makes sense